modules/manage/pages/manage-nodes/manage-data-service-and-rebalance.adoc

```diff
@@ -0,0 +1,32 @@
+= Manage a Data Service and Rebalance
+:description: pass:q[Add or remove Data Service on existing nodes in a cluster and rebalance the cluster.]
+:page-toclevels: 3
+
+[abstract]
+{description}
+
+Add or remove a Data Service on existing nodes in a cluster from the Couchbase UI, REST API, or CLI.
```

Comment:
You can add any service, including the Data Service (kv), when you add or join a new node to a cluster.  Also, when you remove a node, you are removing all the services on that node.  To complete adding or removing a node, rebalance is run.  The xref:manage:manage-nodes/add-node-and-rebalance.adoc[Add a Node and Rebalance] and xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance] describe the process of adding and removing nodes from the cluster using the Couchbase UI, REST API, or CLI.